### PR TITLE
[TCSACR-193] Add nlp 5.0 feature

### PR DESCRIPTION
**Pull request by Inhong**
### Change Description ###

- Add http://tizen.org/feature/nlp feature to dotnet app-filtering.md file.

### API Changes ###

- http://suprem.sec.samsung.net/jira/browse/TCSACR-193

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/472

File: docs/application/dotnet/getting-started/app-filtering.md [hunk 1]
```diff
@@ -227,6 +227,7 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
 | `http://tizen.org/feature/network.wifi.direct.display` | Specify this key, if the application requires the Wi-Fi Direct display feature. | 2.3   |
 | `http://tizen.org/feature/network.wifi.direct.service_discovery` | Specify this key, if the application requires the Wi-Fi Direct service discovery feature. | 2.3   |
 | `http://tizen.org/feature/network.wifi.tdls` | Specify this key, if the application requires the Wi-Fi Tunneled Direct Link Setup (TDLS). | 3.0   |
+| `http://tizen.org/feature/nlp` | Specify this key, if the application requires the Natural Language Processing(NLP) API. | 5.0   |
 | `http://tizen.org/feature/opengles.texture_format.3dc` | Specify this key, if the application requires the 3DC texture format for OpenGL&reg; ES. | 2.2.1 |
 | `http://tizen.org/feature/opengles.texture_format.atc` | Specify this key, if the application requires the ATC texture format for OpenGL&reg; ES. | 2.2.1 |
 | `http://tizen.org/feature/opengles.texture_format.etc` | Specify this key, if the application requires the ETC texture format for OpenGL&reg; ES. | 2.2.1 |
```
Comment (rishu-samsung) [inline docs/application/dotnet/getting-started/app-filtering.md]: At line no. 238, 239, 240 and 241: Please put a space after a period. Suggested edits:

| `http://tizen.org/feature/opengles.version.2_0` | Specify this key, if the application requires OpenGL&reg; ES version 2.0. You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered. | 2.2.1 |

 | `http://tizen.org/feature/opengles.version.3_0` | Specify this key, if the application requires OpenGL&reg; ES version 3.0. You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered. | 2.4   |

 | `http://tizen.org/feature/opengles.version.3_1` | Specify this key, if the application requires OpenGL&reg; ES version 3.1. You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered. | 4.0   |

 | `http://tizen.org/feature/opengles.version.3_2` | Specify this key, if the application requires OpenGL&reg; ES version 3.2. You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered. | 4.0   |

**Comment by mijin85cho** [comment]
Thanks for the review, @rishu-samsung.

Hello, @Inhong, adding to rishu-samsung 's comments,
would you please check the supported profile of `http://tizen.org/feature/nlp`?

It looks like mobile, wearable and TV profiles are supported,
but `http://tizen.org/feature/nlp` is only added to mobile table.

**Comment by mijin85cho** [comment]
Hello, @Inhong. This is reminder comment.
Would you please check the review comments?

**Comment by journey2w** [comment]
@Inhong
There is no ./media/app_filtering_basic_flow.png  file. Could you copy native/tutorials/details/media/app_filtering_basic_flow.png into ./media/? It is same file.

File: docs/application/dotnet/getting-started/app-filtering.md [hunk 2]
```diff
@@ -227,6 +227,7 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
 | `http://tizen.org/feature/network.wifi.direct.display` | Specify this key, if the application requires the Wi-Fi Direct display feature. | 2.3   |
 | `http://tizen.org/feature/network.wifi.direct.service_discovery` | Specify this key, if the application requires the Wi-Fi Direct service discovery feature. | 2.3   |
 | `http://tizen.org/feature/network.wifi.tdls` | Specify this key, if the application requires the Wi-Fi Tunneled Direct Link Setup (TDLS). | 3.0   |
+| `http://tizen.org/feature/nlp` | Specify this key, if the application requires the Natural Language Processing(NLP) API. | 5.0   |
```
[the remark this answers is not in the document]
Comment (Inhong) [reply docs/application/dotnet/getting-started/app-filtering.md]: done

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/472

**Comment by journey2w** [comment]
@Inhong
You can see the following the contents in docs/application/dotnet/getting-started/app-filtering.md.

```
**Figure: Feature-based filtering**

![Feature-based filtering](./media/app_filtering_basic_flow.png)
```

./media/app_filtering_basic_flow.png file means docs/application/dotnet/getting-started/media//app_filtering_basic_flow.png.

**Please move docs/application/media/app_filtering_basic_flow.png to ocs/application/dotnet/getting-started/media//app_filtering_basic_flow.png.**

Moreover, please check Mijin's comment:

```
Thanks for the review, @rishu-samsung.Hello, @Inhong, adding to rishu-samsung 's comments,would you please check the supported profile of http://tizen.org/feature/nlp?It looks like mobile, wearable and TV profiles are supported,but http://tizen.org/feature/nlp is only added to mobile table.
```

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/472

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/472

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**